packages/mds-agency/request-handlers.ts

```diff
@@ -332,9 +285,6 @@ export const submitVehicleEvent = async (req: AgencyApiRequest, res: AgencyApiRe
       return res.status(400).send(failure)
     }
 
-    // make a note of the service area
-    event.service_area_id = getServiceArea(event)
-
```

Comment:
Is `service_area_id` going to remain part of the event and just never be set?

Reply:
For the time being, yeah. We'll need to write a db migration to remove that column from events, but that's not pressing for the time-being I think.